spec/controllers/content_blocks_spec.rb

```diff
@@ -4,6 +4,35 @@
   let!(:content_block) { create(:content_block) }
 
   context 'requesting a content block' do
+    describe 'GET batch_fetch' do
+      # rubocop:disable Metrics/AbcSize
+      def check_object(blocks, index, content_block)
+        object = OpenStruct.new(blocks[index])
+        expect(object.content).to eq(content_block.content)
+        expect(object.content_path).to eq(content_block.content_path)
+        expect(object.api_key).to eq(content_block.api_key)
+        expect(object.id).to be_nil
+      end
+
```

Comment:
yep - autoformatted. i don't like it either... maybe i should simply not enable it anymore for this file?